products/green_fast_track/models/_sources.yml

```diff
@@ -121,6 +151,62 @@ sources:
             tests:
               - not_null
 
+      - name: nysshpo_historic_buildings_polygons
+        columns:
+          - name: usnnum
+            tests:
+              - not_null
+          - name: usnname
+          - name: eligibilitydesc
+            tests:
+            - accepted_values:
+                values: ['Eligible','Listed', 'Not Eligible', 'Not Eligible - Demolished', 'Undetermined']
+          - name: wkb_geometry
```

Comment:
perhaps I missed it somewhere... Can `wkb_geometry` ben null?

Reply:
good call. I don't see any nulls in the source data, will add the test